aspnetcore/getting-started/index.md

```diff
@@ -15,57 +15,56 @@ uid: getting-started
 
 2. Create an ASP.NET Core project. Open a command shell and enter the following command:
 
-    ```console
-    dotnet new webapp -o aspnetcoreapp
-    ```
+   ```console
+   dotnet new webapp -o aspnetcoreapp
+   ```
 
 3. Trust the HTTPS development certificate:
 
 # [Windows](#tab/windows)
 
-    ```console
-    dotnet dev-certs https --trust
-    ```
+  ```console
+  dotnet dev-certs https --trust
+  ```
 
-   The preceding command displays the following dialog:
+  The preceding command displays the following dialog:
 
-   ![Security warning dialog](_static/cert.png)
+  ![Security warning dialog](_static/cert.png)
 
-   Select **Yes** if you agree to trust the development certificate.
+  Select **Yes** if you agree to trust the development certificate.
 
 # [macOS](#tab/macos)
 
-    ```console
-    dotnet dev-certs https --trust
-    ```
+  ```console
+  dotnet dev-certs https --trust
+  ```
 
-   The preceding command displays the following message:
+  The preceding command displays the following message:
 
-   *Trusting the HTTPS development certificate was requested. If the certificate is not already trusted we will run the following command:*
-   `'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <<certificate>>'`
-   *This command might prompt you for your password to install the certificate on the system keychain.
-   Password:*
+  * Trusting the HTTPS development certificate was requested. If the certificate is not already trusted we will run the following command:* `'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <<certificate>>'`.
```

Comment:
This line ... prob others ... could use a bit of UE work, too. I was only looking at indentation on my pass. If you want me to open a UE issue for this and work it, let me know.

Reply:
Sure, I think I have some time over the next few days. Just need some pointers on how I should modify it so I have some guidelines to follow, if possible.

Also, did you intend to put a space between the `*` and `Trusting`? It's now rendering as an unordered list.

Reply:
Oh ....... yeah. Hold on a sec. I'll fix it.